transform-plugins/README.rst

```diff
@@ -102,111 +94,124 @@ JSON Parser
+  **copies:** Specifies the numbers of copies of the input record that are to be emitted.
 
 Stream Formatter
 -------------
-
-:Id:
-  StreamFormatter
+:ID:
+  **StreamFormatter**
 :Type:
   Transform
 :Mode:
   Batch and
   Realtime
 :Description:
-  Formats a structured record as Stream format. Plugin will convert the structured record to Stream format. It will include header configuration and body configuration. The body of Stream event can be of any of the two types : CSV or JSON.
+  Formats a Structured Record as Stream format. Plugin will convert the Structured Record to Stream format.
+  It will include a header and body configurations. The body of the Stream event can be either type CSV or JSON.
 :Configuration:
-    **body:** Specifies the fields from input structured record that should be included in the body of Stream event. 
-    **header:** Specifies the fields from input structured record that should be included in the header of Stream event. 
-    **format:** Specifies the format of the body. Currently supported formats are JSON, CSV, TSV and PSV.
-    **schema:** Specifies the output schema. The output schema can have only two fields. One of type STRING and the other of type MAP<STRING, STRING>.
+  - **body:** Specifies the input Structured Record fields that should be included in the body of the Stream event.
+  - **header:** Specifies the input Structured Record fields that should be included in the header of the Stream event.
+  - **format:** Specifies the format of the body. Currently supported formats are JSON, CSV, TSV, and PSV.
+  - **schema:** Specifies the output schema. The output schema can have only two fields: one of type STRING and the other of type MAP<STRING, STRING>.
 
 Compressor
 -------------
-
-:Id:
-  Compressor
+:ID:
+  **Compressor**
 :Type:
   Transform
 :Mode:
-  Batch and 
+  Batch and
   Realtime
 :Description:
-  Compresses configured fields. Multiple fields can be specified to be compressed using different compression algorithms. Currently plugin supports SNAPPY, ZIP and GZIP types of compression of fields. 
+  Compresses configured fields. Multiple fields can be specified to be compressed using different compression algorithms.
+  Plugin supports SNAPPY, ZIP, and GZIP types of compression of fields.
 :Configuration:
-    **compressor:** Specifies the configuration for compressing fields. Following is how this can be specified in JSON configuration <field>:<compressor>[,<field>:<compressor>]*,
-    **schema:** Specifies the output schema. The fields that are compressed will have the same field name, but they will be of type BYTE array. 
+  - **compressor:** Specifies the configuration for compressing fields. In JSON configuration, this is specified as ``<field>:<compressor>[,<field>:<compressor>]*``.
+  - **schema:** Specifies the output schema. The fields that are compressed will have the same field name but they will be of type BYTE array.
```

Comment:
Is the schema unmodifiable here? If so, there is nothing for the user to configure, right?